blocks/header/header.js

```diff
@@ -153,7 +153,9 @@ function initializeAuth(domain, clientID, audience, responseType, scope) {
 
 // login call
 function login() {
-  webauth.authorize();
+  if (webauth !== undefined) {
```

Comment:
this could be simplified
```suggestion
  if (!webauth) {
```

see https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
Found that we don't need that check anymore. Took it out : 1b1b16a0e74d91d09059846332103ce05861186d